### PR TITLE
Feat/contract size optimize

### PR DESCRIPTION
**Pull request by yuanjunliang**

### CODE REVIEW

**Review by noyyyy — CHANGES_REQUESTED** [review]
git ignore `build` directory

File: src/oz/contracts-upgradeable/utils/CountersUpgradeable.sol
Comment (noyyyy) [inline src/oz/contracts-upgradeable/utils/CountersUpgradeable.sol:1]: delete

File: src/oz/contracts-upgradeable/token/ERC721/utils/ERC721HolderUpgradeable.sol
Comment (noyyyy) [inline src/oz/contracts-upgradeable/token/ERC721/utils/ERC721HolderUpgradeable.sol:1]: delete

File: src/oz/contracts-upgradeable/proxy/ClonesUpgradeable.sol
Comment (noyyyy) [inline src/oz/contracts-upgradeable/proxy/ClonesUpgradeable.sol:1]: delete

File: src/mock/RankMock.sol [hunk 1]
```diff
@@ -26,9 +26,9 @@ contract RankMock is RankUpgradeable {
         return _seasonData[_season]._scoreRank.get(n, 1)[0];
     }
 
-    function setTokenIdToTvlRank(uint256 tokenId) public {
```
Comment (noyyyy) [inline src/mock/RankMock.sol:29]: why comment

**Review by noyyyy — CHANGES_REQUESTED** [review]

File: src/oz/contracts-upgradeable/utils/CountersUpgradeable.sol
Comment (yuanjunliang) [reply src/oz/contracts-upgradeable/utils/CountersUpgradeable.sol:1]: used in draft-ERC20PermitUpgradeable.sol

File: src/RBT.sol [hunk 2]
```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity 0.8.17;
 
-import {ERC20CappedUpgradeable, ERC20Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20CappedUpgradeable.sol";
-import {SafeOwnableUpgradeable} from "@p12/contracts-lib/contracts/access/SafeOwnableUpgradeable.sol";
```
Comment (noyyyy) [inline src/RBT.sol:5]: why  extract `SafeOwnableUpgradeable`. It's our lib package

File: src/oz/contracts-upgradeable/README.md
Comment (noyyyy) [inline src/oz/contracts-upgradeable/README.md:1]: can be deleted

**Review by noyyyy — CHANGES_REQUESTED** [review]

File: src/RBT.sol (hunk 2 above)
Comment (yuanjunliang) [reply src/RBT.sol:5]: SafeOwnableUpgradeable use @openzeeplin  conflect with oz

Comment (noyyyy) [reply src/RBT.sol:5]: get

**Review by noyyyy — APPROVED** [review]

**Pull request merged**